### ISSUE TITLE
Thick focus border during login since firefox 44

### ISSUE THREAD

**Issue by kkretsch** [piwik/piwik#9659]
I'm pretty sure it wasn't until updating from firefox 43.* to 44.0: The piwik login screen now has some strange thick focus border around the input fields. I've seen this at least on two different PCs and two different piwik installations.

**Comment by tsteur** [comment]
Just updated and experience this as well. It was not shown with FF42 & FF43. 

It is because of `:focus { outline: auto;`

@mattab not sure if it is expected or not.

**Comment by mattab** [comment]
This should be fixed in 2.16.0  RC1 see https://github.com/piwik/piwik/pull/9605

**Comment by tsteur** [comment]
It's not.

**Comment by mattab** [comment]
could reproduce after update to FF as well, we need to fix this one before 2.16.0 :+1:
